code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -278,6 +278,8 @@
 			update_icon()
 
 /obj/machinery/smartfridge/drying_rack/accept_check(obj/item/O)
+	if(istype(O, /obj/item/reagent_containers/food/snacks/grown))
+		return TRUE
```

Comment:
This will cause runtimes when putting things that normally don't go inside the drying rack in, since this completely bypasses the check for an item's dried_type.

Reply:
Well what sort of things? I've tried it with all sorts of plants and it works fine.
I mean sure you wouldn't normally smoke a cabbage but there's no harm in it either.

Reply:
I'm looking at the object tree and the only questionable thing that would be included is carpet in plant form.